### ISSUE TITLE
On key down "tab" breaks the carousel

### ISSUE THREAD

**Issue by dominnya** [Domin-MND/portfolio#8]
A bug appears to happen when you use tab key while having your cursor focused on the carousel (not heading). Happens on all browsers:

![An example of bug](https://i.imgur.com/hWKKnSn.png)

**Comment by dominnya** [comment]
TL;DR The fix requires a rewrite to `mantine@^6` while mantine per se got a bug with `viewportProps` property usage.

`mantine@^5` latest does not support `viewportProps` property for ScrollArea.Autosize component causing the bug to appear (link issues were fixed with the `tabindex` attribute). Using a native ScrollArea of mantine would cause unnecessary card section growing to `60vh`. Mantine of the latest version on the other hand has got the type of the `viewportProps` property for Autosize except the actual implementation. Currently waiting for the hot fix update to roll out:
https://github.com/mantinedev/mantine/pull/3760

**Comment by dominnya** [comment]
A mantine PR was successfully merged, the issue was fixed
